### ISSUE TITLE
Desktop Icon missing on Linux

### ISSUE THREAD

**Issue by aplatypus** [defold/defold#8032]
**Describe the bug (REQUIRED)**
I have just installed `Defold` on my PC and for my child's laptop and set up a `Defold.desktop` launcer file -- There is no clear icon in the ZIP file to use for the desktop entry.

**To Reproduce (REQUIRED)**
1. Download the Linux build from the Defold website: `https://defold.com/download`
2. Unpack the ZIP file into a folder, say as:  `/opt/games/Defold`
3. Make a `.desktop` file for the installed app as: `$HOME/Desktop/Defold.desktop`
   * Use executable path: `/opt/apps/Defold/Defold`, as the binary.
   * There is nothing enter for the desktop icon however among the extracted files (I have commented out that entry).
4. Desktop launcher (or menu launcher) shows a default icon from Ubuntu which is more or less meaningless
    * I am setting this up for my daughter and an Icon will help rember which app to fire-up, etc. 
    * And create a positive impression

**Expected behavior (REQUIRED)**
I normally expect to see an icon `.png`, `.jpg`, etc. in the ZIP file.  Otherwise the readme or in the config file you could leave a comment to say where I may download a suitable icon graphic to use.

For comparison, when I used `Flatpak` to install `DBeaver`, it created a  desktop file on my Desktop directory with all that minute resolved.  I think it is a reasonable thing to supply a _model_ `Defold.desktop` file that the user can copy to the `$HOME/Desktop/` directory.

**Defold version (REQUIRED):**
  - Version **1.5.0**

**Platforms (REQUIRED):**
 - Platforms: Linux 
 - OS: Kubuntu 23.04
 - Device: PC and Laptop PC

**Minimal repro case project (OPTIONAL):**
N/A

**Logs (OPTIONAL):**
N/A

**Workaround (OPTIONAL):**

* I would be satisfied with a link on the download webpage to say:  "use this `.png` graphic for a desktop icon".

**Screenshots (OPTIONAL):**
N/A

**Additional context (OPTIONAL):**

*  **Sample `Defold.desktop` file**

```
[Desktop Entry]
Categories=Game;Development;
Comment=High-performance cross-platform games
Exec=/opt/apps//Defold/Defold
GenericName=3D Game Engine
##  Icon=/opt/apps/Defold/Defold
Keywords=3D;Gaming;
Name=Defold
StartupNotify=true
Type=Application
Version=1.5
```

**Comment by britzl** [comment]
There is an app icon inside the jar file, but that is not very convenient in this case. We'll look into changing the package script to also put an icon in the zip archive root.

**Comment by aplatypus** [comment]
* Just a thought, if the icon is in the JAR then it ought to be in a resources folder someplace in the source tree, wouldn't it?
* Can I find the `.png` file or whatever on Github?

**Comment by aplatypus** [comment]
* Icon found in project for those looking ...
  * _see_: [bundle-resources](https://github.com/defold/defold/tree/dev/editor/bundle-resources), in `editor`

**Comment by vlaaad** [comment]
I believe this issue is now solved with #8095. We now bundle the icon in the distribution, and there is now a `Help -> Create Desktop Entry` command that installs the Defold `.desktop` entry.

**Comment by aplatypus** [comment]
:-)  Many thanks
